Tests/LibWeb/Text/input/Compression/round-trip.html

```diff
@@ -22,7 +24,7 @@
                 offset += result.value.byteLength;
             }
 
-            let result = new Uint8Array(buffer, 0, offset).toBase64();
+            let result = new TextDecoder().decode(new Uint8Array(buffer, 0, offset));
```

Comment:
Since you've removed evidence of the stream actually being changed by the compressor, maybe add a check that the bytes are actually different during the roundtrips?

Reply:
I've improved the test substantially.